src/meta/src/hummock/manager/mod.rs

```diff
@@ -1512,6 +1521,9 @@ where
                 *compaction_group_id,
             );
         }
+        for (compaction_group_id, num_levels) in deleted_compaction_groups {
```

Comment:
It's possible meta restarts between `commit_multi_var` and  `remove_compaction_group_in_sst_stat`?

Reply:
> It's possible meta restarts between `commit_multi_var` and `remove_compaction_group_in_sst_stat`?

yes, but are you willing to check the latest `VersionDelta` in `load_meta_store_impl()`?

Reply:
> It's possible meta restarts between `commit_multi_var` and `remove_compaction_group_in_sst_stat`?

fixed